File: services/java-spring/test_service/services/check/CheckService.java
package test_service.services.check;

import java.math.BigDecimal;
import java.time.*;
import java.util.*;

import test_service.models.*;

public interface CheckService {
	void checkEmpty();
	CheckForbiddenResponse checkForbidden();
	SameOperationNameResponse sameOperationName();
}

File: services/scala-play/services/services/CheckService.scala
package services

import javax.inject._
import scala.concurrent._
import services.check._
import models._

@Singleton
class CheckService @Inject()()(implicit ec: ExecutionContext) extends ICheckService {
  override def checkEmpty(): Future[Unit] = Future { ??? }
  override def checkForbidden(): Future[CheckForbiddenResponse] = Future { ??? }
  override def sameOperationName(): Future[SameOperationNameResponse] = Future { ??? }
}
